stackslib/src/net/api/postblock_proposal.rs

```diff
@@ -300,12 +333,18 @@ impl NakamotoBlockProposal {
         let size = builder.get_bytes_so_far();
         let cost = builder.tenure_finish(tenure_tx)?;
 
+        println!("block header: {:?}", block.header);
+        println!("expected: {:?}", self.block.header);
+
```

Comment:
Drop

Reply:
Fixed in 1fad8d226e8081b4204c6a6ecc44c5ff8038013b